tests/utils.py

```diff
@@ -23,6 +23,42 @@ def numpy_installed():
 go_dir = os.path.join(thisdir, "test-recipes", "go-package")
 archive_dir = os.path.join(thisdir, "archives")
 
+#Moved From test_api_debug.py file
+recipe_path = os.path.join(metadata_dir, "_debug_pkg")
+ambiguous_recipe_path = os.path.join(metadata_dir, "_debug_pkg_multiple_outputs")
+tarball_path = os.path.join(thisdir, "archives", "test_debug_pkg-1.0-0.tar.bz2")
+
+def switch_between_shell_and_bash():
+	if on_win:
+		shell_cmd = ["cmd.exe", "/d", "/c"]
+	else:
+		shell_cmd = ["bash", "-c"]
+	return shell_cmd
```

Comment:
since this is always going to be the same value for the session we should use an immutable global instead:

```suggestion
SHELL_CMD = ("cmd.exe", "/d", "/c") if on_win else ("bash", "-c")
```

Reply:
why was this marked as resolved?

Reply:
oooh I'll change that.. I mistook the suggestion